splash/tests/test_jsonpost.py

```diff
@@ -157,6 +157,36 @@ def test_cookie(self):
         self.assertStatusCode(r, 200)
         self.assertIn("bar", r.text)
 
+    def test_http_POST_request_from_splash(self):
+        formbody = {"param1": "one", "param2": "two"}
+
+        r = self.request({
+            "url": self.mockurl("postrequest"),
+            "formdata": formbody,
+            "body": urllib.urlencode(formbody),
```

Comment:
Why are both formdata and body passed here? If formdata is supported it could make sense to raise an error, if it is not supported removing it will make the test more clear.